app/main/views.py

```diff
@@ -20,6 +20,11 @@ def index():
     ]), 200
 
 
+@main.route('/_status')
+def status():
+    return jsonify(status="ok")
```

Comment:
Is there any more useful information we could add to this? Can we get at current build number? Or some info about the beanstalk instance/environment to identify exactly which app is up?